### PR TITLE
Spotify ESB Connector

### PR DESCRIPTION
**Pull request by Shakila**

### CODE REVIEW

File: redmine/redmine-connector/redmine-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/redmine-issueCategories/deleteIssueCategory.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <parameter name="reassignToId"
+               description="If issues are assigned to the category, this helps to reassign these issues to the category with this id."/>
+    <sequence>
+        <property name="uri.var.issueCategoryId" expression="$func:issueCategoryId"/>
+        <property name="uri.var.reassignToId" expression="$func:reassignToId"/>
+        <filter xpath="(string(get-property('uri.var.reassignToId'))) or get-property('uri.var.reassignToId') != ''">
+        <then>
+            <property name="uri.var.optionalParam" expression="fn:concat('?reassign_to_id=',get-property('uri.var.reassignToId'))" />
+        </then>
+    </filter>
```
Comment (keerthu) [inline redmine/redmine-connector/redmine-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/redmine-issueCategories/deleteIssueCategory.xml]: Formatting issues.

File: spotify/src/main/resources/connector.xml [hunk 2]
```diff
@@ -1,16 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (keerthu) [inline spotify/src/main/resources/connector.xml]: Update the licence.

Comment (Shakila) [inline spotify/src/main/resources/connector.xml]: Already updated in current code: https://github.com/wso2/esb-connectors/blob/master/spotify/spotify-connector/spotify-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/connector.xml

File: redmine/redmine-connector/redmine-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/redmine-issueCategories/deleteIssueCategory.xml (hunk 1 above)
Comment (Shakila) [inline redmine/redmine-connector/redmine-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/redmine-issueCategories/deleteIssueCategory.xml]: Resolved in :https://github.com/wso2/esb-connectors/pull/382

**Pull request merged**